script.js

```diff
@@ -0,0 +1,61 @@
+var name = prompt("Hi there, what's your name?");
+var year = prompt("Nice to meet you "+name+", what year would you like to go to?");
+
+for(var i = year; i <= 2015; i++) {
+name = "Calvin Klein";
+}
```

Comment:
This loop isn't really 'doing' anything except for setting name to `Calvin Klein` (assuming that year is 2015 or earlier). If that's what you intend to do, and if statement would be more clear.